<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_ui</class>
 <widget class="QDialog" name="login_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 16, 84);
font: 16pt &quot;Chakra Petchl&quot;;</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameField">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 14pt &quot;Arial&quot;;
    color: rgb(0, 0, 0);
    border: 2px solid rgb(26, 16, 84);
    border-radius: 15px;  
padding: 2px</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordField">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 14pt &quot;Arial&quot;;
    color: rgb(0, 0, 0);
    border: 2px solid rgb(26, 16, 84);
    border-radius: 15px;  
padding: 2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 16, 84);
font: 16pt &quot;Chakra Petchl&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

    color: white;
    background-color: rgb(26, 16, 84);
    border-radius: 15px;
    padding: 2px;
}

/* Pressed state â†’ white background */
QPushButton:pressed {
    
	color: rgb(26, 16, 84);
    background-color: white;
    border: 2px solid rgb(26, 16, 84);
    border-radius: 15px;
    padding: 2px;
}
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo_holder">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
